.run/intellij-rust.github.io.run.xml

```diff
@@ -0,0 +1,35 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="intellij-rust.github.io" type="docker-deploy" factoryName="dockerfile" server-name="Docker">
+    <deployment type="dockerfile">
+      <settings>
+        <option name="imageTag" value="intellij-rust.github.io" />
+        <option name="containerName" value="intellij-rust.github.io" />
+        <option name="portBindings">
+          <list>
+            <DockerPortBindingImpl>
+              <option name="containerPort" value="8080" />
+              <option name="hostIp" value="0.0.0.0" />
+              <option name="hostPort" value="8080" />
+            </DockerPortBindingImpl>
+          </list>
+        </option>
+        <option name="sourceFilePath" value="Dockerfile" />
+        <option name="volumeBindings">
```

Comment:
As I understand correctly, volumes here are needed to be able to apply changes made locally without rerunning configuration. But why only `_posts` and `assets/posts/` directories? What if I want to change some other places? Maybe it's better to add volume for the whole project directory instead of copying its files during docker image building?